docs/Concepts/Privacy/Privacy-Overview.md

```diff
@@ -4,7 +4,7 @@ description: Privacy
 # Privacy
 
 Privacy in Pantheon refers to the ability to keep transactions private between the involved parties. 
-Other parties cannot access the transaction content, sending party, or list of participating parties. 
+Other parties cannot access the transaction content, sending party, or list of participating parties. All parties in a private transaction must be online when the transaction is sent.  
```

Comment:
I'd probably include this as a second sentence in the Important box. 

This is subtle but I'd probably avoid using the word _sent_.  The private transaction is submitted via the eea_sendRawTransaction JSON-RPC API, distributed to participant's Orion nodes, the privacy marker tx generated and submitted. So it's not clear which part of those steps _sent_ covers.  Suggest something like: 
All private transaction participants must be online for a private transaction to be successfully distributed.  If any participants are offline when the private transaction is submitted, the transaction fails and must be resubmitted.